packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaForm.tsx

```diff
@@ -225,7 +231,12 @@ const TableColumnSchemaForm = React.forwardRef<
 
   const addDefaultColumns = useCallback(() => {
     if (defaultColumnModels) {
-      addColumnSet(defaultColumnModels)
+      addColumnSet(
+        defaultColumnModels.map(cm => ({
+          ...cm,
+          id: undefined,
```

Comment:
For some reason, including the `id` broke SWC's RPC call to create and mutate the columns. Omit the ID to fix this.